docs/android/best_practices.md

```diff
@@ -38,6 +38,8 @@ Logging is essential but should be used judiciously. As Jake Wharton says in his
 
 When working with time, date, or duration, avoid using primitive types. Instead, use strong types to prevent unit mix-ups.
 
+:::note[Example]
```

Comment:
Now you should probably change the headings inside this block to level 4 as well for consistency and to avoid them showing up in the sidebar